examples/load_save_configuration.py

```diff
@@ -0,0 +1,48 @@
+from ingeniamotion import MotionController
+
+
+def main() -> None:
```

Comment:
I think this example could be confusing for someone who wants to load and save a config. It seems more like a test than an example. What do you think?

Reply:
I agree, a basic example is necessary, but I'm going to keep the current example as well. The reason is posted in the description issue.

Reply:
Done!